src/Sarif.UnitTests/Visitors/SarifVersionOneToCurrentVisitorTests.cs

```diff
@@ -1,194 +1,69 @@
-﻿//// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
-//// license. See LICENSE file in the project root for full license information.
```

Comment:
>//// [](start = 0, length = 4)

NOTE TO SELF: prefer multi-line C style code commenting in future to minimize code diffs when re-enabled. :P